robots/solo_description/meshes/obj/solo12/solo_12_base.mtl

```diff
@@ -0,0 +1,9 @@
+# File produced by Open Asset Import Library (http://www.assimp.sf.net)
+# (assimp v5.0.0)
+
+newmtl DefaultMaterial
```

Comment:
Why do we need to add all the `.mtl` files?

The looks like control configuration, we are not adding any reference about controllers here.

Reply:
`.mtl` is for adding material/color to the meshes when visualising

Reply:
PDF & png files are already here and not part of this PR, so this is a separate issue.

the `12` option is already in the loader script.

Reply:
Thanks :)